src/components/resumes/ResumesAPI.js

```diff
@@ -64,3 +65,25 @@ export async function getResume(id) {
     return { data, handlerError };
   }
 }
+
+export async function getSkillsByName(name) {
+  let handlerError = false;
+  const data = {};
+  try {
+    const { data } = await axios.get(
+      `${API_HOST}/api/${API_VERSION}/${API_NAME_SKILLS}?name=${name}`
```

Comment:
can you add a variable for url:
const url = `${API_HOST}/api/${API_VERSION}/${API_NAME_SKILLS}?name=${name}`;
and you use only url;

Reply:
Done